config.yml

```diff
@@ -11,7 +11,7 @@ params:
     disableLanguageSwitchingButton: true
     disableAssetsBusting: true
     editURL: "https://github.com/sourcebots/docs/tree/master/content/"
-    ordersectionsby: "title"
+    ordersectionsby: "weight"
```

Comment:
Does this fall back to 'title' for all sections with the same weight?

Reply:
Yes. It was just a hack to move the FAQs section to the bottom of the list